lib/transport_web/api/controllers/datasets_controller.ex

```diff
@@ -33,6 +33,7 @@ defmodule TransportWeb.API.DatasetController do
 
   defp transform_dataset(dataset) do
     %{
+      "id" => dataset.datagouv_id,
```

Comment:
maybe name it `datagouv_id` in case we want to add `id` after

Reply:
:+1: fixed